src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static boolean diff_string_check(String[] arr, String toCheckValue)
+    {
```

Comment:
This seems to be a violation of the [coding standard](https://se-education.org/guides/conventions/java/index.html#layout), more specifically you should use K&R style brackets. I noticed the same issue in several places below too.